src/Microsoft.Health.Fhir.Shared.Api/Modules/OperationsModule.cs

```diff
@@ -53,6 +53,8 @@ public void Load(IServiceCollection services)
             services.Add<ReindexJobWorker>()
                 .Singleton()
                 .AsSelf();
+
+            services.AddFactory<IScoped<IAnonymizer>>();
```

Comment:
Perhaps keep things together in the AnonymizerModule

Reply:
Thanks, updated.